src/components/App/AppLoadingProgressBar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Line } from 'rc-progress'
```

Comment:
I choose `rc-progress` mostly because it was a super light svg implementation, but thinking about it, we could likely just use a `div` with background colour and width set 👍.